pandas/core/config_init.py

```diff
@@ -307,6 +307,48 @@ def mpl_style_cb(key):
     return val
 
 
+def table_schema_cb(key):
+    # Having _ipython_display_ defined messes with the return value
+    # from cells, so the Out[x] dicitonary breaks.
```

Comment:
actually i would create this dynamically
simply define 2 methods and have the cb just set one of them

we do in set_use_numexpr just like this